src/cmds/pbs_ralter.c

```diff
@@ -294,7 +294,8 @@ main(int argc, char *argv[], char *envp[])		/* pbs_ralter */
 			pbs_server, pbs_errno);
 		CS_close_app();
 		exit(pbs_errno);
-	}
+	} else
+		show_svr_inst_fail(connect, argv[0]);
```

Comment:
why are we showing that a server instance failed if connect > 0 ?

Reply:
Now that I look at the function, I see that it checks whether any server instance was not successfully connected to. Can you please change the function name? It's kind of misleading. Maybe something like "check_msvr_conn_fail" ?

Reply:
It does more than just checking. The important function is to write to stderr upon failure. How about show_svr_inst_fail_if_any() ?

Reply:
no that's too ugly. See my suggestion for cnt2server(), if you agree then this function can just be for checking while the client does the printing of error if needed.

Reply:
Right now we are more specific on which of the instance is down. We don't print the port number now but I am going to add that. It will be helpful in printing the specifics than leaving a generic connection error.